Sources/Unbox.swift

```diff
@@ -85,28 +85,39 @@ public func Unbox<T: UnboxableWithContext>(data: NSData, context: T.ContextType,
 
 // MARK: - Error type
 
+public enum InvalidInputError: ErrorType, CustomStringConvertible {
```

Comment:
I'd name this `UnboxValueError` to indicate that these are errors related to the **values** that are being unboxed (rather than other types of input like `NSData` or a closure, for example). What do you think?